### PR TITLE
Update eeprom.md

### PR DESCRIPTION
**Pull request by V205Github**
## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)
- Fixed extra ; and added some notes. Changed tags.

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.
Thanks.

### CODE REVIEW

File: content/hardware/02.hero/boards/uno-r4-minima/tutorials/eeprom/eeprom.md [hunk 1]
```diff
@@ -14,18 +13,21 @@ In this tutorial you will learn how to access the EEPROM (memory) on an **Arduin
 The goals of this tutorials are:
 
 - Write to the EEPROM memory,
-- read from the EEPROM memory.
+- Read from the EEPROM memory.
 
 ## Hardware & Software Needed
 
-- Arduino IDE ([online](https://create.arduino.cc/) or [offline](https://www.arduino.cc/en/main/software))
+- [Arduino IDE](https://www.arduino.cc/en/main/software)) or ([Arduino Create](https://create.arduino.cc/)
```
Comment (jacobhylen) [inline content/hardware/02.hero/boards/uno-r4-minima/tutorials/eeprom/eeprom.md:20]:
```suggestion
- Arduino IDE ([online](https://create.arduino.cc/) or [offline](https://www.arduino.cc/en/main/software))
```

File: content/hardware/02.hero/boards/uno-r4-minima/tutorials/eeprom/eeprom.md [hunk 2]
```diff
@@ -14,18 +13,21 @@ In this tutorial you will learn how to access the EEPROM (memory) on an **Arduin
...
+- USB-C cable to your computers USB port type(often USB-C or USB-A).
```
Comment (jacobhylen) [inline content/hardware/02.hero/boards/uno-r4-minima/tutorials/eeprom/eeprom.md:21]:
```suggestion
- USB-C cable 
```

**Review by jacobhylen — APPROVED** [review]
LGTM!
Thank you @V205Github!

**Comment by V205Github** [comment]
Thank you @jacobhylen !

**Pull request merged**